hyp3_autorift/geometry.py

```diff
@@ -89,7 +90,7 @@ def bounding_box(safe, priority='reference', polarization='hh', orbits='Orbits',
     return lat_limits, lon_limits
 
 
-def polygon_from_bbox(lat_limits, lon_limits):
+def polygon_from_bbox(lat_limits: Tuple[float, float], lon_limits: Tuple[float, float]) -> ogr.Geometry:
```

Comment:
unrelated to this PR, but I've found myself migrating towards always ordering it as `lon, lat` to match the gdal 3 convention

Reply:
It's pretty up in the air for which is better -- GDAL mostly does `lon, lat` but not always, PROJ calls the lat-long projection `latlon`, but returns stuff in lon, lat order, [ISO says lat, lon](https://stackoverflow.com/a/51148140). 

Since we mostly use GDAL, it's probably better to do lon, lat, but I don't think I'll tackle that here.